test/command/run_command_integration_test.dart

```diff
@@ -5,14 +5,15 @@ import 'package:test/test.dart';
 
 void main() {
   group("run", () {
-    
     test("should notify if no projects found", () async {
-      final emptyDirectory = "${Directory.current.path}/test/command/test_project/empty_directory";
+      final emptyDirectory =
+          "${Directory.current.path}/test_resources/command/test_project/empty_directory";
```

Comment:
Do we need `command` directory? Can we  move `test_project` and `test_project_fo_test` one directory up?

Reply:
Currently, there's a 1-1 relation of the directory layout in tests and production code if those tests are in `command` directory.